src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -284,5 +258,57 @@ internal static FunctionLoadResponse FunctionLoadRequestHandler(FunctionLoadRequ
 
             return response;
         }
+        internal static FunctionEnvironmentReloadResponse EnvReloadRequestHandler(FunctionEnvironmentReloadRequest request, WorkerOptions workerOptions)
+        {
+            var envReloadResponse = new FunctionEnvironmentReloadResponse
+            {
+                Result = new StatusResult { Status = StatusResult.Types.Status.Success }
+            };
+
+            if (AppContextUtils.IsRunningWithNativeHost())
+            {
```

Comment:
I'm concerned about this logic here. This layer shouldn't behave differently based on whether we're running in the native worker. The behavior here should be consistent and all of the differences scoped to the native client instead. We want to make sure we don't have discrepancies between the two models.